include/browseview_func.php

```diff
@@ -133,24 +133,24 @@ function categories_func( $atts )
 	</script>
 <?php
 	$display .= "<div id=searchresults>";
-	$semnumber = rtrim(str_replace(".", "-", $_REQUEST["semnumber"]), "-");
+	$semdomain = isset($_REQUEST["semdomain"]) ? trim($_REQUEST["semdomain"]) : '';
```

Comment:
Not a requirement, but could be simplified to 
`$semdomain = trim($_REQUEST["semdomain"] ?? '');`

Reply:
Also, I see that `$_GET` is used later in this file, so the preferred method would probably be like this:
`$semdomain = trim((string)filter_input(INPUT_GET, 'semdomain', FILTER_UNSAFE_RAW));`

Reply:
Nice! I was not aware of filter_input. I should have done more research. I was getting tired of doing isset($_GET['xxx']) and was thinking of writing my own function, but got lazy... Thanks!